default.nix

```diff
@@ -49,4 +52,19 @@ pkgs.stdenv.mkDerivation {
     "fortify"
     "fortify3"
   ];
+
+  # Running Hyrise executables requires the LD_LIBRARY_PATH variable set. This does not
+  # happen by other hooks, therefore it happens manually here. Each of the derivations
```

Comment:
Looking at the nix documentation, I wonder why you call those items "derivations" here. Those are not "specification[s] for running an executable".

Reply:
Technically, the packages in the list are derivations. But I guess that confuses people more than it helps here. I will change it!

Reply:
I understand why you use them (although I am surprised nix doesn’t handle that). I am wondering about your usage of this term derivation.

Reply:
Technically but not according to the nix documentation?